java-client/src/main/java/org/scassandra/http/client/PrimingClient.java

```diff
@@ -185,7 +191,7 @@ private void httpDelete(String url) {
         }
     }
 
-    private void prime(PrimingRequest primeRequest, String url) {
+    private void prime(Object primeRequest, String url) {
```

Comment:
It isn't exposed to the client. I wanted to avoid them sharing an interface from an API point of view as i didn't want it possible to pass in a batch prime to the prime method.